storage/rocksdb/ha_rocksdb.cc

```diff
@@ -417,6 +433,8 @@ static char *rocksdb_strict_collation_exceptions;
 static my_bool rocksdb_collect_sst_properties = 1;
 static my_bool rocksdb_force_flush_memtable_now_var = 0;
 static my_bool rocksdb_force_flush_memtable_and_lzero_now_var = 0;
+static my_bool rocksdb_enable_ttl = 1;
```

Comment:
And we have a winner:
`static unsigned long rocksdb_persistent_cache_size_mb;`
...
`static MYSQL_SYSVAR_ULONG(
    persistent_cache_size_mb, rocksdb_persistent_cache_size_mb,
    PLUGIN_VAR_RQCMDARG | PLUGIN_VAR_READONLY,
    "Size of cache in MB for BlockBasedTableOptions::persistent_cache "
    "for RocksDB", nullptr, nullptr, rocksdb_persistent_cache_size_mb,
    /* min */ 0L, /* max */ ULONG_MAX, 0);
`
<<< Using uninitialized static for the default value in the macro. This value is not explicitly set anywhere in the code. This may explain some of the instability I have with the persistent_cache test.